underfs/hdfs/src/main/java/alluxio/underfs/hdfs/AlluxioHdfsException.java

```diff
@@ -32,42 +32,43 @@
 public class AlluxioHdfsException extends AlluxioRuntimeException {
 
   /**
-   * Converts an AmazonClientException to a corresponding AlluxioHdfsException.
+   * Converts an Hdfs exception to a corresponding AlluxioHdfsException.
    *
-   * @param cause hdfs exception
+   * @param e hdfs exception
    * @return alluxio hdfs exception
    */
-  public static AlluxioHdfsException from(Exception cause) {
-    cause = convertException(cause);
-    Status status = getStatus(cause);
-    // Almost all HDFS exception are not retryable
-    return new AlluxioHdfsException(status, cause.getMessage(), cause, false);
-  }
-
-  private AlluxioHdfsException(Status status, String message, Throwable cause, boolean isRetryAble,
-      Any... details) {
-    super(status, message, cause, isRetryAble, details);
-  }
-
-  private static Status getStatus(Exception e) {
-    //Map response status
-    final Status s;
+  public static AlluxioHdfsException from(Exception e) {
+    e = convertException(e);
+    Status s;
+    ErrorType type;
     if (e instanceof SecurityException) {
       s = Status.PERMISSION_DENIED;
+      type = ErrorType.User;
```

Comment:
Style nits: you can just `return new AlluxioHdfsException(Status.PERMISSION_DENIED, e.getMessage(), e, ErrorType.User);` here. Code will be shorter and you don't need `else`